docs/usage/advanced.rst

```diff
@@ -95,3 +95,13 @@ it is enough to set log level to the desired value.
     logging.basicConfig()
     root_logger = logging.getLogger()
     root_logger.setLevel(logging.DEBUG)
+    
+Get a count of entities using a getattr() function
```

Comment:
This is too specific and in wrong doc section. 

This example belongs to the basic querying doc page - for example directly after first subsection. Ignore the count since it is over-specific example, the main focus is the getattr() usage.

Since this is referencing the issue Getting an EntitySetProxy #165, the intent of the user, in similar fashion described there is better as subtitle - e.g. "Get one entity referenced dynamically" or similar

Reply:
I agree with too specific title -> the title does not describe what was achieved, it describes the technical solution but users in the need for the solution does not know the technical solution so they cannot use the title.

I do not  agree with the objection about a wrong section as this definitely belongs to the Advanced section and the querying is used just to show what can be done. The title could be "Dynamically referenced EntitySet" ?

Reply:
OK, I was seeing it as simple usage of one of [python built-ins](https://docs.python.org/3/library/functions.html) for something pyodata has no fluent API - just the addressing of the entitysets like in the first query user will ever read in docs; querying.rst the first "howto practicaly use pyodata" after "how to initialize". Everything in advanced (Batch, Errors, Logging) are topics unrelated to each other and something that users may want to read after their first working queries - s so that was my rationale. 

But I agree the second point is not that important as the first about the too specific title and can be in both .rst files; just that there is something to reference in future issues :)